app/ui/styles/modal-dialog-custom-styles.d.ts

```diff
@@ -0,0 +1,2 @@
+export = css;
```

Comment:
I don't see a reason why we need type definitions for style sheets. Remove type definitions for styles.